renderer/src/components/Totals/Totals.tsx

```diff
@@ -177,7 +180,7 @@ const Totals = ({ selectedDate }) => {
       <Hint
         learningMethod="buttonClick"
         order={2}
-        groupName="totals"
+        groupName={`${HINTS_GROUP_NAMES.TOTALS}`}
```

Comment:
Why are you formatting HINTS GROUP_NAMES.TOTALS string into a string? There are a lot of places in your code when you're using string literals and formatting strings into strings. 
I think it should be removed from the code.

Reply:
`groupName={HINTS_GROUP_NAMES.TOTALS}`

Reply:
>Why are you formatting HINTS GROUP_NAMES.TOTALS string into a string?
Because I'm a bit of an idiot. Thanks for noticing)